storage/database/interface.go

```diff
@@ -86,6 +86,7 @@ func WriteBatches(batches ...Batch) (int, error) {
 			if err := batch.Write(); err != nil {
 				return 0, err
 			}
+			batch.Reset()
```

Comment:
While making tests, I found it is more rationale to call `batch.Reset()` here, since `batch.Write()` means it is done with the batch. 

As the batches are not reused after calling `WriteBatches()`, it still have the same behavior as now.